packages/next/src/build/webpack/plugins/define-env-plugin.ts

```diff
@@ -133,6 +133,7 @@ export function getDefineEnv({
         }),
     'process.turbopack': isTurbopack,
     'process.env.TURBOPACK': isTurbopack,
+    'process.env.NEXT_RSPACK': process.env.NEXT_RSPACK || '',
```

Comment:
prefer to use a variable like `process.env.__NEXT_BUNDLER` to determine the actual bundler here rather than writing the condition on the UI side.